sysfs/fixtures/devices/platform/applesmc.768/fan1_input

```diff
@@ -0,0 +1 @@
+0
```

Comment:
Appears that some sensors fixtures made their way into this PR.  Can you remove these?